contrast_scw.py

```diff
@@ -46,6 +46,8 @@ def map_contrast_lang_to_scw_lang(contrast_lang):
 
 # Loop through all the Assess rules
 for rule in contrast.list_org_policy(org_id, org_key):
+    print('Processing rule: ' + rule['name'] + '/' + rule['title'])
```

Comment:
for future ref you can use python f strings like:
`print(f'Processing rule: {rule['name']}/{rule['title']}')`